Framework/Algorithms/src/SampleCorrections/MayersSampleCorrectionStrategy.cpp

```diff
@@ -342,7 +338,7 @@ double MayersSampleCorrectionStrategy::sigmaTotal(const double flightPath,
  * @return The associated TOF value
  */
 double MayersSampleCorrectionStrategy::tof(const size_t i) const {
-  return m_histogram ? 0.5 * (m_tof[i] + m_tof[i + 1]) : m_tof[i];
+  return m_histogram.points()[i];
```

Comment:
This makes things slower: If `m_histogram` is storing `BinEdges`, `points()` will now be recomputed once for every `i` (the whole vector!). You should buffer the points in the class to avoid this.